api/reboot_prosemirror/v1/authority.proto

```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
 
-package rbt.thirdparty.prosemirror.v1;
+package reboot_prosemirror.v1;
```

Comment:
Why the package change? Plan was to move this into Reboot `thirdparty` once we get stability. Can we keep it please? Thanks!

Reply:
There is a fix for the associated issue on the way. I'll wait for `0.18.1` before landing this.